### PR TITLE
remove expand button and unread messages badge

### LINKED ISSUE #469: [1] Disable expand option for the widget
To reproduce:

- detach widget
- close widget
- widget can't be reopened

### PR DESCRIPTION
**Pull request by ear-dev**
resolves https://github.com/WideChat/Rocket.Chat/issues/469  and  https://github.com/WideChat/Rocket.Chat/issues/492

### CODE REVIEW

**Comment by ear-dev** [comment]
@Shailesh351 Can you take a look at this please?  I added the build files, but just want to make sure it looks right to you. Thanks.

File: src/components/Screen/index.js [hunk 1]
```diff
@@ -94,13 +94,6 @@ class ScreenHeader extends Component {
 							}
 						</Header.Action>
 					</Tooltip.Trigger>
-					{(!expanded && !windowed) && (
-						<Tooltip.Trigger content={I18n.t('Expand chat')} placement='bottom-left'>
-							<Header.Action aria-label={I18n.t('Expand chat')} onClick={onOpenWindow}>
-								<OpenWindowIcon width={20} height={20} />
-							</Header.Action>
-						</Tooltip.Trigger>
-					)}
```
Comment (Shailesh351) [inline src/components/Screen/index.js]: @ear-dev We should comment it instead of removing. Will help in future if we want to reenable it.

Comment (ear-dev) [reply src/components/Screen/index.js]: @Shailesh351 Initially I tried commenting it out, but I could not get it to work.  As if comments are not allowed in this kind of html block?  Let me know if you have a suggestion.  Thanks.

Comment (Shailesh351) [reply src/components/Screen/index.js]: @ear-dev We can actually do like below,
```
{/* Visasat : Hide Expand Button */}
{/* {(!expanded && !windowed) && (
	<Tooltip.Trigger content={I18n.t('Expand chat')} placement='bottom-left'>
		<Header.Action aria-label={I18n.t('Expand chat')} onClick={onOpenWindow}>
			<OpenWindowIcon width={20} height={20} />
		</Header.Action>
	</Tooltip.Trigger>
)} */}
```

**Pull request merged**